src/plugin.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "OpenNMS Plugin for Grafana",
-  "id": "opennms-helm-app",
+  "id": "grafana-plugin-app",
```

Comment:
I feel we should probably use `opennms-grafana-plugin-app`. We should have `opennms` in the ID, so as to identify the plugin as ours